its/src/test/java/com/sonar/it/csharp/Tests.java

```diff
@@ -30,6 +30,8 @@
 import java.util.List;
 import javax.annotation.CheckForNull;
 import javax.annotation.Nullable;
+
```

Comment:
Empty line (is it intentional?)

Reply:
Most probably not, I will fix it in an upcoming PR.